client/action.py

```diff
@@ -75,5 +75,5 @@ def execute(self, s: 'State', variables):
     'NoOp',
     lambda agt, agtFrom: [Atom('AgentAt', agt, agtFrom)],
     lambda agt, agtFrom: [Atom('AgentAt', agt, agtFrom)],
-    lambda agt, agtFrom: [Atom('AgentAt', agt, agtFrom)],
+    lambda agt, agtFrom: [Atom('AgentAt', agt, agtFrom), Atom('Free', agtFrom)],
```

Comment:
Why does it free the cell if the agent is still there?

Reply:
It is one of the negative effects, so its ¬Free(agtFrom). But maybe AgentAt should be removed as a negative effect

Reply:
But right now NoOp is not being applied as an action, so it shouldn't effect the rest of the code yet...